src/main/java/vexatos/tgregworks/integration/TGregRegistry.java

```diff
@@ -78,7 +78,7 @@ public void registerToolParts() {
 		for(Materials m : toolMaterials) {
 			toolMaterialNames.add(m.mDefaultLocalName);
 			int matID = getMaterialID(m);
-			TConstructRegistry.addToolMaterial(matID, m.name(), m.mDefaultLocalName, m.mToolQuality,
+			TConstructRegistry.addToolMaterial(matID, m.name(), m.mLocalizedName, m.mToolQuality,
```

Comment:
This field does not appear to exist on GregTech 5.07.07, so it won't build like this.

Reply:
mDefaultLocalName will only return english name, instead of localized name.

Reply:
mDefaultLocalName will only return english name, instead of localized name.